spec.md

```diff
@@ -127,6 +130,25 @@ message ValidateCredentialResponse {
 
 `ValidateCredential` is a REQUIRED operation which verifies that the credential provided to the application maps to a known subject, such as a JWT with a valid signature and expiry in the future. If the credential is valid, implementations MUST respond with `result` set to `RESULT_VALID` and `subject` populated accordingly. Otherwise, implementations MUST respond with `result` set to `RESULT_INVALID`.
 
+##### `GetAccessToken`
```

Comment:
I'm not sure this should be in the Authentication service. Per the spec, the service "manages verification of credentials provided to applications". This is different from _getting_ an access token, which should probably be its own service.